crates/ruff_linter/src/rules/eradicate/detection.rs

```diff
@@ -26,7 +26,7 @@ static HASH_NUMBER: Lazy<Regex> = Lazy::new(|| Regex::new(r"#\d").unwrap());
 static POSITIVE_CASES: Lazy<RegexSet> = Lazy::new(|| {
     RegexSet::new([
         // Keywords
-        r"^(?:elif\s+.*\s*:|else\s*:|try\s*:|finally\s*:|except\s+.*\s*:)$",
+        r"^(?:elif\s+.*\s*:|else\s*:|try\s*:|finally\s*:|except\s+.*\s*:|case\s+.*\s*:.*)$",
```

Comment:
note: this group ends with `.*` to detect the one-line case:
```python
case "hi": print()
```

Reply:
I feel like the other cases don't catch that -- can you try to add a test case with `try` followed by a one-line body?

Reply:
Yep - I think this will catch more non-code statements though like "to try: use iterators" or something. I'll write some tests to see

Reply:
Ok seems to work well, should I add `.*` for all of the cases and tests?

Reply:
Let's remove the trailing `.*` from this PR, and we can open a separate PR to add `.*` to all of the cases. It'll make it easier to approve and merge these as separate changes.

Reply:
Cool will do

Reply:
Ah sorry, I meant the opposite: can we remove the `.*` from here, so that we don't support one-line for _any_ of the patterns (like before this PR), and then open a separate PR that adds the `.*` to all of them at once?

Reply:
Yep, that's what I understood you to mean!

Reply:
Done! Ticket opened, I'll open that PR next.

Reply:
> Yep, that's what I understood you to mean! I'll do that and open an issue to add .* in another PR

(Oops, sorry, I think I was looking at the code, and thought you'd already pushed!)